src/playermat/PlayermatApi.ttslua

```diff
@@ -406,7 +406,14 @@ do
     for guid, pos in pairs(storedPositions) do
       local obj = getObjectFromGUID(guid)
       obj.setPosition(mat.positionToWorld(pos))
-      obj.setRotation(obj.getRotation():setAt("y", rotationY))
+
+      -- offset the rotation by 180 degrees if the guid matches the player hand zones
+      local finalRotationY = rotationY
+      if guid == "0285cc" or guid == "a70eee" or guid == "5fe087" or guid == "be2f17" then
+        newRotationY = rotationY - 180
```

Comment:
```suggestion
        finalRotationY = rotationY + 180
```
Seems like you used the wrong variable name? Also I am not sure if TTS handles negative rotations well 😅